core/rendered_connection.ts

```diff
@@ -362,6 +362,8 @@ export class RenderedConnection
         aria.setState(highlightSvg, aria.State.LABEL, 'Open connection');
       }
     }
+
+    this.sourceBlock_.recomputeAriaLabel();
```

Comment:
It seems like a lot of extra work to recompute the entire aria label which includes searching through all of the inputs, child blocks, etc. just to remove one specific ID for the connection highlight, which we already know the ID of.

Could you keep track of the childElemIds as a set and just remove or add this connection ID to the set, and then update the aria-owns attribute with the set of IDs instead of recalculating the entire set of IDs on each highlight?